### PR TITLE
MLDB-2050 auto id use _ instead of -

### PR DESCRIPTION
**Pull request by FinchPowers**

### CODE REVIEW

**Comment by FinchPowers** [comment]
Anyone knows if the doc needs to be updated?

File: testing/MLDB-983-create-dataset-ids.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var dataset = mldb.createDataset(config);
 mldb.log(config);
 mldb.log(dataset.id());
 
-if (config.id.indexOf("auto-") != 0)
+if (config.id.indexOf("auto_") != 0)
     throw "ID should start with 'auto-' : '" + config.id + "'";
```
Comment (guyd) [inline testing/MLDB-983-create-dataset-ids.js]: auto- -> auto_

**Comment by guyd** [comment]
+1

**Comment by FinchPowers** [comment]
"make autotest" passes locally.

**Pull request merged**